OracleHTTPServer/README.md

```diff
@@ -95,6 +60,31 @@ _This volume will be created in "/var/lib/docker" directory or the location wher
 
 ######Later you can login to running container and configure Weblogic Server proxy plugin file and run *restartOHS.sh* script.
 
+### Stopping and re-starting OHS instance
+To stop the OHS instance, execute the following steps:
+1. Exec into the container and execute /bin/bash (Assuming the name of container is 'ohs')
+
+      docker exec -it ohs /bin/bash
+
+2. Navigate to /u01/oracle/ohssa/oracle_common/common/bin
+
+      cd /u01/oracle/ohssa/oracle_common/common/bin
+
+3. Execute stop-ohs.py 
+ 
+      ./wlst.sh /u01/oracle/container-scripts/stop-ohs.py
+
+
+To re-start the OHS instance, execute the following steps:
```

Comment:
I have defined a SIGTERM handler in my code. That invokes stop-ohs.py when a SIGTERM is received by the container. I will remove this description from README.md